packages/color-block/src/ColorBlockDropsView.tsx

```diff
@@ -0,0 +1,127 @@
+                    </div>
+
+                    <div className="tw-flex tw-w-[100px] tw-mb-3 tw-text-m tw-font-bold tw-text-black tw-text-center">
+                        <RichTextEditor designTokens={designTokens ?? undefined} readonly={!isEditing} />
+                    </div>
+
+                    {colorSpaces?.map((colorSpaceID: string) => {
+                        const mappedColorSpace = mapColorSpaces(colorSpaceID);
+
+                        return (
+                            <div key={colorSpaceID} className="tw-flex tw-items-center tw-mb-1 last:tw-mb-0">
+                                <div className="tw-mr-1 tw-text-s tw-text-black-70">{mappedColorSpace.value}</div>
+
+                                {!isEditing ? (
+                                    <Tooltip
+                                        withArrow
+                                        position={TooltipPosition.Right}
+                                        hoverDelay={0}
+                                        content={
+                                            <>
+                                                Color Name <br />
+                                                #100100 <br />
+                                                <span className="tw-text-black-50">Click to copy</span>
+                                            </>
+                                        }
+                                        triggerElement={
+                                            <div
+                                                className="tw-relative tw-z-[1] tw-cursor-pointer tw-text-s tw-text-black-80"
+                                                onClick={() => copyToClipboard('test')}
+                                            >
+                                                {mappedColorSpace.placeholder}
+                                            </div>
+                                        }
+                                    />
+                                ) : (
+                                    <div className="tw-text-s tw-text-black-80">{mappedColorSpace.placeholder}</div>
+                                )}
+                            </div>
+                        );
+                    })}
+                </div>
+            ))}
+
+            {isEditing && (
+                <div className="tw-flex tw-flex-col tw-items-center tw-w-1/6">
+                    <ColorsBlockColorPicker onSelect={(value) => console.log(value)}>
+                        <div className="tw-relative tw-z-[1] tw-flex tw-justify-center tw-items-center tw-w-[100px] tw-h-[100px] tw-mx-auto tw-cursor-pointer tw-rounded-full tw-mb-3 tw-text-black tw-bg-[#F1F1F1]">
+                            <IconPlus size={IconSize.Size24}></IconPlus>
+                        </div>
+                    </ColorsBlockColorPicker>
+
+                    <div className="tw-flex tw-w-[100px] tw-mb-3 tw-text-m tw-font-bold tw-text-black tw-text-center">
```

Comment:
maybe try to reuse/ extract some classNames or even components. It will later simlify it when you want to adapt some styles across all three views. 

e.g. `const COLORNAME_CLASSNAME = "tw-flex tw-w-[100px] tw-mb-3 tw-text-m tw-font-bold tw-text-black tw-text-center"`, `<div className={COLORNAME_CLASSNAME}`

or even the whole component like:
``` const ColorName: FC = () => (
                    <div className="tw-flex tw-w-[100px] tw-mb-3 tw-text-m tw-font-bold tw-text-black tw-text-center">
 <RichTextEditor
                            designTokens={designTokens ?? undefined}
                            readonly={!isEditing}
                            placeholder="Color Name"
                        />
 )
 </div>```